### ISSUE TITLE
Checklist for release 1.0 - minor tasks

### ISSUE THREAD

**Issue by aaronkaplan** [certtools/intelmq#608]
## Collection task list for minor issues for release 1.0

Here we collect small tasks which need to be handled before we can release 1.0
- [x] set loglevel to INFO by default
- [x] check for deprecations and remove them
- [x] when dropping Message.update in favor of dict.update: implement sanitation (as in `__init__`).
- [x] Set `error_log_exception` and `error_log_message` to `false`

**Comment by sebix** [comment]
- [ ] check for deprecations and remove them

**Comment by aaronkaplan** [comment]
how?

**Comment by sebix** [comment]
We could use master for either the current release or as development branch ("bleeding edge"). In case of the second, all PRs should then go to 'develop' which is the default branch then. Otherwise it's the other way round: master is bleeding-edge (where development happens) and we have release branches.

**Comment by dmth** [comment]
Regarding Sebix' comment: https://github.com/certtools/intelmq/issues/608#issuecomment-242185306 I offered to share a branching strategy with you. I failed to do so. Let's split this discussion into a new Issue.
